actions/bump-version/action.yaml

```diff
@@ -5,6 +5,10 @@ inputs:
   release-type:
     description: "The type of the release (major, minor or patch)."
     required: true
+  working-directory:
+    description: "The root directory of the poetry project."
```

Comment:
I don't think this necessarily refers to a root of a poetry project. Just use a similar description like you did in the Readme